modules/nextflow/src/test/groovy/nextflow/script/ScriptIncludesTest.groovy

```diff
@@ -149,7 +149,7 @@ class ScriptIncludesTest extends Dsl2Spec {
         result.val == 'dlrow olleh'
     }
 
-    def 'should not allow duplicate functions' () {
+    def 'should allows duplicate functions' () {
```

Comment:
Let's add a bit more tests, e.g. the invocation of `foo()` and at least another test with a more complex signature e.g. a string + a optional map

Reply:
done